apps/experiments/model_audit_fields.py

```diff
@@ -33,3 +33,6 @@
     "confirmation_text",
     "team",
 ]
+
+TEAM_FIELDS = ["name", "slug", "members"]
+MEMBERSHIP_FIELDS = ["team", "user", "role"]
```

Comment:
A bit odd to have these in the experiments app. Maybe move them to the `teams` app?

Reply:
Lol, fair

Reply:
https://github.com/dimagi/open-chat-studio/pull/215/commits/6ed76de0e0efc6cd0806de2429245e535b91fa5d